### DISCUSSION TITLE
第6节课作业

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#472] [作业]
作业贴在这

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture6.md

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/homework/homework_6.md

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/tree/Assignment

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/yo7mkqp3alm4rk5z?singleDoc#

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture6-b64ca08312d34be0990a9d7c3189da23?pvs=4

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/71768615-5968-458a-91e6-0e3c61036735
